xml/System/String.xml

```diff
@@ -8063,11 +8070,17 @@ A string is empty if it  is explicitly assigned an empty string ("") or <xref:Sy
         </Parameter>
       </Parameters>
       <Docs>
-        <param name="separator">To be added.</param>
-        <param name="values">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="separator">The character to use as a separator. <paramref name="separator" /> is included in the returned string only if <paramref name="value" /> has more than one element.</param>
+        <param name="values">An array of objects from which their string representations will be concatenated.</param>
+        <summary>Concatenates the string representations of an array of objects, using the specified separator between each member.</summary>
+        <returns>A string that consists of the elements of <paramref name="values" /> delimited by the <paramref name="separator" /> character.
+
+-or-
+
+<see cref="F:System.String.Empty" /> if <paramref name="values" /> has no elements or <paramref name="separator" /> and all the elements of <paramref name="values" /> are <see cref="F:System.String.Empty" />.</returns>
```

Comment:
>or <paramref name="separator" /> and all the elements of <paramref name="values" /> are <see cref="F:System.String.Empty" [](start = 83, length = 121)

This doesn't look correct. 

doing String.Join(",", new object[] {"", "", "", "" }) returns string like ",,," and not empty string

Reply:
ignore this. I missed the part that the separator is empty too.

---
In reply to: [328368295](https://github.com/dotnet/dotnet-api-docs/pull/3238#discussion_r328368295) [](ancestors = 328368295)